examples/plot_structure_basic/plot_inferred_structure.py

```diff
@@ -0,0 +1,223 @@
+"""
+Plot an inferred 3-d structure with matplotlib
+==============================================
+
+This example showcases how to plot an inferred 3-d structure output by
+PASTIS using matplotlib.
+"""
+
+import numpy as np
+import itertools
+from mpl_toolkits.mplot3d import axes3d
+import matplotlib
+import matplotlib.pyplot as plt
+import pandas as pd
+from scipy import interpolate
+
+##############################################################################
+# Get a dataframe for the inferred structure's coordinates.
+df = pd.read_csv('data/struct_inferred.000.coords', delimiter=" ", header=None)
+
+
```

Comment:
Your code here is not quite pep8. Can you run `flake8` on it and check that all the formatting is fine? Just right here, you have too many blank lines between the two functions.

Reply:
Oh my - flake8 revealed quite a few problems, mainly with whitespace. I will fix them